api/oneclick.py

```diff
@@ -272,6 +281,66 @@ def release_hold(self, patron, pin, licensepool):
             patron_oneclick_id, item_oneclick_id)
 
 
+    def update_licensepool_for_identifier(self, isbn, availability):
+        """Update availability information for a single book.
+
+        If the book has never been seen before, a new LicensePool
+        will be created for the book.
+
+        The book's LicensePool will be updated with current approximate 
+        circulation information (we can tell if it's available, but 
+        not how many copies). 
+        Bibliographic coverage will be ensured for the OneClick Identifier. 
+        Work will be created for the LicensePool and set as presentation-ready.
+
+        :param isbn the identifier OneClick uses
```

Comment:
for OneClick ebooks and eaudio, it's always an isbn.  The only other ids they have are for magazines, which aren't accessible to us right now.